crates/apub/src/protocol/activities/community/update.rs

```diff
@@ -19,4 +26,27 @@ pub struct UpdateCommunity {
   #[serde(rename = "type")]
   pub(crate) kind: UpdateType,
   pub(crate) id: Url,
+  pub(crate) audience: Option<ObjectId<ApubCommunity>>,
+}
+
+#[async_trait::async_trait(?Send)]
+impl InCommunity for UpdateCommunity {
```

Comment:
This seems like its repeated a lot, could maybe be a default trait impl. Can worry about it later tho.